cmd/kubeadm/app/cmd/init.go

```diff
@@ -316,6 +311,13 @@ func newInitData(cmd *cobra.Command, args []string, options *initOptions, out io
 		return nil, err
 	}
 
+	ignorePreflightErrorsSet, err := validation.ValidateIgnorePreflightErrors(options.ignorePreflightErrors, cfg.NodeRegistration.IgnorePreflightErrors)
```

Comment:
Ideally we should call this twice - once for command line supplied errors and once for config file supplied ones. This is because of the "all" keyword (it must not be specified in the config file).
After that, we should have a couple of sets, that need merging. If the command line set has "all" in it, the resulting set should be consisting only of `{"all"}`. Otherwise it's a simple merge of both. Probably a helper func should do the trick.

Reply:
@rosti  if config has something specific, and command line says `ignore all` we still need to fail. We shouldn't allow to mix 'all' with anything else. Let admin think twice (or more times) when admin tries to mask all errors.

Reply:
This indeed sounds like a desirable behaviour to me.